src/core/linter-rules/check-punctuation.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Linter rule "check-punctuation". Makes sure the there are no punctuations missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "check-punctuation";
+const punctuationMarks = [".", ":", "!", "?"];
+
+const meta = {
+  en: {
+    description: `<p> tags should end with either of ${punctuationMarks} `,
```

Comment:
Please confirm this is rendering properly. The punctuationMarks will be missing spaces, no?

Reply:
This is what is showing 

![screenshot from 2018-04-03 07-49-31](https://user-images.githubusercontent.com/25258877/38225704-bcbb42de-3713-11e8-8bd7-7dace2856936.png)

Reply:
Ok, I thought it might :) Fixing -. I forgot to mention that `description` and `howToFix` are markdown.